src/EditorFeatures/VisualBasicTest/UseIsNullCheck/UseIsNullCheckForReferenceEqualsTests.vb

```diff
@@ -10,11 +10,11 @@ Imports Roslyn.Test.Utilities
 Imports Xunit
 
 Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.UseIsNullCheck
-    Partial Public Class UseIsNullCheckTests
+    Partial Public Class UseIsNullCheckForReferenceEqualsTests
```

Comment:
why don't we need a VB version?

Reply:
VB has already had an idiomatic way to do this check forever.  i.e. ```foo is nothing``` or ```foo isnot nothing```.  Because this was always the way to do this check in VB, we don't ahve the same need to look for people doing some alternate that we would want them to update.

Reply:
I agree but given that we already have the ReferenceEquals version for VB as well, it would make sense to be consistent